lsst_efd_client/efd_helper.py

```diff
@@ -43,7 +46,7 @@ class EfdClient:
 
     def __init__(self, efd_name, db_name='efd',
                  creds_service='https://roundtable.lsst.codes/segwarides/',
-                 client=None):
+                 client=None, timeout=None):
```

Comment:
Should we pick a timeout that we know will work in most cases, since we know of some cases where the underlying default does not?

Reply:
Good idea, I'll use 900s for the default `aiohttp.client` timeout.